### DISCUSSION TITLE
Vscode workbench startup editor

### DISCUSSION THREAD

**Discussion by sandeepsharmadevops** [eclipse-theia/theia#12603] [General] [answered]
Team,

Please share any references on how to have vscode workbench startup editor feature.

Thanks

**Comment by msujew** [comment]
Hey @sandeepsharmadevops, do you mind explaining a bit what you mean with the *vscode workbench startup editor* feature? It doesn't ring a bell and Google doesn't give me any results either.

**Comment by sandeepsharmadevops** [comment]
Hey, @msujew vscode workbench startup editor allows managing the landing page to be opened during vscode launch time, hence looking how to have the same when leveraging theia.

For reference:
![Screenshot 2023-06-09 at 6 00 48 PM](https://github.com/eclipse-theia/theia/assets/12079189/7f3b75a5-89e8-47ba-a60a-62ea254ecf72)

Let me know for anything else.

Thanks!!

**Reply by msujew** [reply, marked as answer]
I see. You can probably just implement a `FrontendApplicationContribution` that opens one of these widgets on startup in the `onStart` method. See also [this answer on our discourse forum](https://community.theia-ide.org/t/having-a-widget-open-by-default/2330/2).
